src/lib/serviceRoutines/putIndividualContextEntity.cpp

```diff
@@ -39,6 +40,22 @@
 /* ****************************************************************************
 *
 * putIndividualContextEntity - 
+*
+* Corresponding Standard Operation: UpdateContext/UPDATE
+*
+* PUT /v1/contextEntities/{entityId::id}
+* PUT /ngsi10/contextEntities/{entityId::id}
+*
+* Payload In:  UpdateContextElementRequest
+* Payload Out: UpdateContextElementResponse
+*
+* URI parameters:
+*   - attributesFormat=object
```

Comment:
Sure, I'll add a few more phases of the func-test.
And no, not for POST, this is a PUT operation. POST will come later.